src/index.js

```diff
@@ -88,10 +171,10 @@ async function getPackages (ghToken, repoName, repoOwner, packageName, packageCo
   // get just a list of versions and the begin and end of the list
   pkgDescList = pkgDescList.map(buildVersions)
   pkgAscList = pkgAscList.map(buildVersions)
+  // Smashing the two objects together
   const fullList = [...new Set([...pkgDescList, ...pkgAscList])]
   const keepList = fullList.filter(({ id: id1 }) => pkgDescList.some(({ id: id2 }) => id1 === id2))
```

Comment:
have we verified that the list of images/objects that come back are kept in order now? I know we're adding, we'll potentially adding, way more things to the list now

Reply:
I will double check that the `asc` and `desc` orders are kept!